applications/ila-cookie-banner/RELEASE.md

```diff
@@ -0,0 +1,94 @@
++ Click on the the latest successful run - the top one with a green icon.
+
+	> Tip: Failed runs are not expected. Contact the development team if the most recent `Cookie banner zip` GitHub action failed (indicated by a red icon).
+
++ Find and click `cookie-banner` under the `Artifacts` section of the `GitHub Actions Cookie banner zip` page.
++ Find the Zip file in your browser downloads directory.
+	
+### Step 3. Extract and place the contents of the zip file into the folder created in step one, above.
+
+After extracted correctly, the contents of the `1.0.0` folder should look like:
+
+	```
+	1.0.0
+	├── css
+	│   ├── ila-cookie-banner.css
+	│   ├── ila-cookie-uic-colors.css
+	│   ├── ila-cookie-uis-colors.css
+	│   └── ila-cookie-uiuc-colors.css
+	├── js
+	│   └── ila-cookie-banner.js
+	├── otSDKStub.js
+	├── partials
+	│   └── ila-cookie-banner-content.part.html
+	└── RELEASE.md
+	```
+
+### Step 4. Verify the main JavaScript file
+
++ With a text editor, open `otSDKStub.js`.
++ For release `1.0.0`, verify that the first line of `otSDKStub.js` contains `1.0.0`:
+	
+	```
+	let cookie_url = 'https://onetrust.techservices.illinois.edu/1.0.0';
+	```
+	
+	> Tip: If the version number (such as `1.0.0`) in the first line does not match the expected release version, reach out to the development team.
+	
+## Production Release
+
+### Step 5. Activate the New Release
+
++ Make a backup copy of `https://onetrust.techservices.illinois.edu/otSDKStub.js` to `YYYY.MM.DD.otSDKStub.js`
+
+> NOTE: **The next step causes the new version of the cookie banner to start appearing across all sites that use the file.**
+
++ Put the new `otSDKStub.js` at the CDN root. - Copy the file verified in step 3 - `https://onetrust.techservices.illinois.edu/1.0.0/otSDKStub.js` to the CDN root `https://onetrust.techservices.illinois.edu/otSDKStub.js`.
+
+> Tip: You can roll back to the previous version by restoring `YYYY.MM.DD.otSDKStub.js` to `otSDKStub.js`. This is recommended if any part of `Step 6: Verify the Release` fails.
+
+### Step 6. Verify the release
+
++ In a browser, visit a website that imports `https://onetrust.techservices.illinois.edu/otSDKStub.js` with a `<script src=...>` line.
+
+> For example: https://cookieme.rslater.web.illinois.edu/
+
++ Press F12 to open the `Developer Tools`.
++ Find and open the `Network` tab of `Developer Tools`.
++ Ensure that `Disable Cache` is checked. (Otherwise the browser may re-use old downloaded files to save time, but break this test.)
++ Reload the page, to capture a copy of all network traffic.
++ Review the rows returned - find the rows from our CDN domain (`onetrust.techservices.illinois.edu`).
+
+> Tip: If no such rows are found, then the deployment was not fully successful. Restore `YYYY.MM.DD.otSDKStub.js` to `otSDKStub.js` and reach out to the development team.
+
++ In these rows, confirm that the expected release number (i.e. `1.0.0`) appears in the `GET URL` details.
+
+For example:
```

Comment:
This should probably just be part of the bullet above. The examples also need to be indented.